### PR TITLE
Themes implementation

### PR DESCRIPTION
**Pull request by ProvoK**
Related to https://github.com/magmax/python-inquirer/issues/29

It's still in developing, just a couple of hours work. I would like to start a conversation to talk about the `Theme` attributes and overall design.

### CODE REVIEW

**Comment by magmax** [comment]
I like the approach, but it should have a builder in order to allow to use it from the json approach...

The problem here is that the output is changing with the default theme, and it shouldn't. It should be exactly the same.

And fix Travis, of couse.

**Comment by ProvoK** [comment]
Hello there! 
> The problem here is that the output is changing with the default theme, and it shouldn't. It should be exactly the same.

Are you talking about the default value in brackets and the selected values shown inline?
I added the default value in brackets taking inspiration from here  https://github.com/AlecAivazis/survey. If you agree, i can implement this behaviour as optional.
The latter thing, that is selected values shown inline, i agree can be useless for Checkbox or List and i can drop that behaviour.

Tonight i'll continue working on the feature, i'll start adding the json/dict loader :)

**Comment by ProvoK** [comment]
@magmax Hello, tonight i did some work. There is still one error with python2, but i'm too tired now, i'll continue later. When you can give me a feedback on the progression :)

File: inquirer/prompt.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # -*- coding: utf-8 -*-
 
 from .render.console import ConsoleRender
+from . import themes
 
 
 def prompt(questions, render=None, answers=None,
-           raise_keyboard_interrupt=False):
-    render = render or ConsoleRender()
+           theme=themes.Default(), raise_keyboard_interrupt=False):
```
Comment (magmax) [inline inquirer/prompt.py]: Beware tabs!

**Comment by magmax** [comment]
looks fine

**Comment by ProvoK** [comment]
finally 🎉  
asap i'll add documentation and some tests :)

**Comment by ProvoK** [comment]
@magmax Sorry for this load of commits. I finally implemented some tests. I don't know how to contribute with docs

**Comment by magmax** [comment]
It uses sphinx. Just see files under docs directory. They are written in RST (Restructured Text) format. If you open them from github, you will see them rendered.

I think that editing usage.rst should be enough. Will you try?

**Comment by ProvoK** [comment]
i thought it was more difficult, i should have done it :)

**Comment by ProvoK** [comment]
@magmax let me know id there is something else to do :)

**Comment by ProvoK** [comment]
@magmax Hi, will you merge this?

**Comment by magmax** [comment]
Yes, I will, but I need some time to test it and release it.

Please, give me until the next week.

**Pull request merged**